src/main/java/com/mkdk72ki/asgmt10/mapper/UserMapper.java

```diff
@@ -19,4 +21,12 @@ public interface UserMapper {
 
     @Select("SELECT * FROM users WHERE id = #{id}")
     Optional<User> findById(int id);
+
+    // POST
+    @Select("SELECT id From users WHERE email = #{email}")

```

Comment:
SQLの予約語は大文字で統一しましょうか。
```suggestion
    @Select("SELECT id FROM users WHERE email = #{email}")

```

Reply:
572cd057a57b2215fe97394d703000a3635fa6f0 対応しました。  
ご確認お願いいたします。